### DISCUSSION TITLE
where is log file?

### DISCUSSION THREAD

**Discussion by xiuer-1573** [alvr-org/ALVR#1015] [Q&A]
Log file output in what directory and how to change the log level, I see a macro is debug!  How do I set up debug!  Output to logs?  thank you

**Comment by zmerp** [comment]
You can enable log to disk in settings->extra tab. The file is session_log.txt in the installation root on Windows, and in ~/.config/alvr on Linux (I think). You can set the log level always in settings->extra tab, advanced mode

**Reply by xiuer-1573** [reply]
thank you very much
